src/ukify/ukify.py

```diff
@@ -1080,10 +1080,8 @@ def pe_add_sections(opts: UkifyConfig, uki: UKI, output: str) -> None:
 
                 encoded = json.dumps(j).encode()
                 if len(encoded) > section.SizeOfRawData:
-                    raise PEError(
-                        f'Not enough space in existing section .pcrsig of size {section.SizeOfRawData} to append new data of size {len(encoded)}.'  # noqa: E501
-                    )
-
+                    raise PEError(f'Not enough space in existing section .pcrsig of size {section.SizeOfRawData} '
+                                  f'to append new data of size {len(encoded)}')  # noqa: E501
```

Comment:
This is what CI chokes on:
```suggestion
                    raise PEError(
                        f'Not enough space in existing section .pcrsig of size {section.SizeOfRawData} to append new data of size {len(encoded)}'  # noqa: E501
                    )
```
To quote you: "Don't split error messages." ;)